libobs/obs-scene.c

```diff
@@ -401,13 +401,17 @@ static void calculate_bounds_data(struct obs_scene_item *item,
 
 	width = (float)(*cx) * scale->x;
 	height = (float)(*cy) * scale->y;
-	width_diff = item->bounds.x - width;
-	height_diff = item->bounds.y - height;
+	// Disregards flip when calculating size diff
```

Comment:
In c files it is suggested to /* Comment */ styled comments, instead of //

Reply:
That is only required for multiline comments. For single line comments, there is no set rule for `//` vs. `/* */`.